qa/L0_python-self-test/test.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 # used pip packages
-pip_packages="nose numpy opencv-python pillow"
+pip_packages="nose numpy opencv-python pillow torch"
```

Comment:
I'd move this tests to something pytorch specific. The motivation to do that is that we don't want to repeat the whole `L0_python-self-test` for every version of `torch` supported, since most of the test cases don't use `torch`